src/cpu/aarch64/jit_uni_eltwise.cpp

```diff
@@ -19,11 +19,14 @@
 #include "common/bfloat16.hpp"
 #include "common/c_types_map.hpp"
 #include "common/dnnl_thread.hpp"
+#include "common/eltwise_pd.hpp"
 #include "common/nstl.hpp"
+#include "common/type_helpers.hpp"
 #include "common/utils.hpp"
 
 #include "cpu/aarch64/jit_generator.hpp"
 
+#include <cstdint>
```

Comment:
Usually system headers have a different grouping, preferably at the bottom of all other headers included.